charts/longhorn/100.1.4+up1.2.6/Chart.yaml.orig

```diff
@@ -0,0 +1,28 @@
+apiVersion: v1
```

Comment:
Not sure why this chart.yaml.orig appear in hear. Looks like we need to rerun the make prepare, make patch, and make chart

Reply:
I double check the files and execute `make prepare`, 'make patch` and `make charts` to re-generate charts again, and the `chart.yaml.orig` was not generated.

Reply:
This is just a temp file due to the previous failed make patch u did, but u just didn't notice it.

Simply deleting it from your comment, because it's not needed.

Reply:
Looking good now. Thank you